api/src/main/java/au/org/aodn/nrmn/restapi/validation/process/RawValidation.java

```diff
@@ -71,7 +71,7 @@
                                                                 ValidationLevel.BLOCKING)),
                                 Tuple2.of("Block",
                                                 new IntegerFormatValidation(StagedRow::getBlock, "Block",
-                                                                Arrays.asList(0, 1, 2, 10))),
+                                                                Arrays.asList(0, 1, 2))),
```

Comment:
This is for ATRC and RLS - does the same rule apply to both?

Reply:
Will check with @bpasquer

Reply:
Yes the rule applies to both